lib/Server.js

```diff
@@ -302,19 +302,28 @@ function Server(compiler, options) {
 	}, this);
 
 	if(options.https) {
-		// for keep supporting CLI parameters
-		if(typeof options.https === "boolean") {
+		// pfx support
+		if(options.pfx) {
```

Comment:
Ah, I now understand what you mean. I wasn't sure that was going to work but I tried it and it worked. I've updated the PR with the change. Thanks